### PR TITLE
New api userdata

### PR DESCRIPTION
**Pull request by moul**

### CODE REVIEW

**Comment by QuentinPerez** [comment]
LGTM

File: skeleton-common/usr/local/bin/oc-userdata [hunk 1]
```diff
@@ -9,27 +9,25 @@ USERDATA_URL=${USERDATA_URL:-"http://${USERDATA_IP}/user_data"}
 
 get() {
     URL=$1
-    if type curl 2>/dev/null; then
+    if type curl >/dev/null 2>/dev/null; then
```
Comment (aimxhaisse) [inline skeleton-common/usr/local/bin/oc-userdata]: &> /dev/null

Comment (moul) [inline skeleton-common/usr/local/bin/oc-userdata]: Not working, I don't know why

File: skeleton-common/usr/local/bin/oc-userdata [hunk 2]
```diff
@@ -9,27 +9,25 @@ USERDATA_URL=${USERDATA_URL:-"http://${USERDATA_IP}/user_data"}
...
+	echo "'curl' dependency is missing." >&2
     fi
-    echo "$BODY"
 }
 
 patch() {
     URL="$1"
     DATA="$2"
-    if type curl 2>/dev/null; then
+    if type curl >/dev/null 2>/dev/null; then
```
Comment (aimxhaisse) [inline skeleton-common/usr/local/bin/oc-userdata]: same

**Comment by aimxhaisse** [comment]
lgtm, not important comments

File: skeleton-common/usr/local/bin/oc-userdata (hunk 2 above)
Comment (moul) [inline skeleton-common/usr/local/bin/oc-userdata]: still not working, I don't know why

**Pull request merged**